src/routes/routes.ts

```diff
@@ -0,0 +1,47 @@
+import { Router, Request, Response, NextFunction } from "express";
+import { SEARCH_WEB_COOKIE_NAME } from "../config/config";
+import { searchController as advancedSearchController } from "../controllers/advanced-search/index.controller";
+import downloadController from "../controllers/advanced-search/download.controller";
+import { searchController as alphabeticalSearchController } from "../controllers/alphabetical-search/index.controller";
+import { searchController as dissolvedSearchController } from "../controllers/dissolved-search/index.controller";
+import * as pageUrls from "../model/page.urls";
+import * as templatePaths from "../model/template.paths";
+import { BasketLink, getBasketLink } from "../controllers/utils/utils";
+import { mapPageHeader } from "../utils/page.header.utils";
+import uuid = require("uuid/v4");
+import Cookies = require("cookies");
+
+const router = Router();
+
+/**
+ * Simply renders a view template.
+ *ß
```

Comment:
not sure this character should be in the comment?

Reply:
I'll remove that! 👍